targets/TARGET_STM/TARGET_STM32F7/TARGET_STM32F767xI/TARGET_UHURU_RAVEN/uhuru_raven_init.h

```diff
@@ -0,0 +1,37 @@
+/* mbed Microcontroller Library
+* Copyright (c) 2019 Uhuru Corporation
```

Comment:
For new files, can you add SPDX idenfier please?

SPDX-License-Identifier: Apache-2.0 in this file, for some ST related, they are BSD-3-Clause